tests/test_geometry.py

```diff
@@ -312,6 +326,20 @@ def test_region_contains_same_entities():
     assert region == expected_region
 
 
+def test_region_get_parent():
+    pocket = mc.get_region("rotor pocket")
+    expected_region = mc.get_region("rotor")
+
+    assert pocket.get_parent(mc) == expected_region
+
+
+def test_region_get_children():
```

Comment:
This test is failing with the new version. I have updated the file a bit so can you take a look at this? 
![image](https://github.com/ansys/pymotorcad/assets/80680946/bce5eb38-3400-4361-8273-73d3c42e8300)

Reply:
Region names not unique. Is this a problem?

Reply:
No, not an issue, test file is using compatibility settings for unique naming, test shows it is doing the correct thing so happy to leave as is

Reply:
Issue fixed